src/main/java/uk/gov/companieshouse/officerfiling/api/service/FilingDataServiceImpl.java

```diff
@@ -223,15 +226,19 @@ public Boolean mapCorporateDirector(Transaction transaction, AppointmentFullReco
         return false;
     }
 
-    void setTm01DescriptionFields(FilingApi filing, OfficerFilingData officerFilingData, AppointmentFullRecordAPI companyAppointment) {
+    private void setTm01DescriptionFields(FilingApi filing, OfficerFilingData officerFilingData, AppointmentFullRecordAPI companyAppointment) {
         final String formattedTerminationDate = LocalDate.ofInstant(officerFilingData.getResignedOn(), ZoneOffset.UTC).format(formatter);
         filing.setDescriptionIdentifier(TM01_FILING_DESCRIPTION);
+        var surname = "";
+        var middleNames = "";
         var officerFilingName = "";
         if (companyAppointment.getSurname() != null) {
-            officerFilingName = getNameFromAppointment(companyAppointment);
+            surname = companyAppointment.getSurname().toUpperCase();
+            middleNames = companyAppointment.getOtherForenames();
+            officerFilingName = companyAppointment.getForename() + " " + middleNames + " " + surname;
         } else {
             // is a corporate director
-            officerFilingName = companyAppointment.getName().toUpperCase();
+            officerFilingName = companyAppointment.getName();
```

Comment:
Not clear why changes in this method are being made?

Reply:
Yea had merge issues yesterday but now reverted changes to main